pygame/math.py

```diff
@@ -0,0 +1,1529 @@
+""" Math module """
+
```

Comment:
You've done the vast majority of the work on math.py, so add whichever names you think are relevant.

For the test cases, we've been leaving largely unchanged, so leaving math_test.py as is seems fine.